static/js/components/SiteComponent.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react"
```

Comment:
did you run into an issue with doing `import React from 'react'` ? just curious

Reply:
I'm not sure why I added it but it doesn't appear to break anything to import only the default export